core/constants/content_types.py

```diff
@@ -47,7 +47,7 @@ class ContentTypes(Constant):
 }
 
 
-def get_ct_id(ct_name: str) -> int:
+def get_ct_id(ct_name: str) -> ContentType:
```

Comment:
Nice catch @alashow. Could you actually update the `ct_id = ...` line, so that it gets the id rather than the ContentType instance)?

i.e., `ct_id = ContentType.objects.get_by_natural_key(app_label=app_label, model=model).id`

Reply:
If the current version works, should we even change it? The function is mostly used to pass ids in queryset filters, and it seems to be the filters can correctly use given models (I'm guessing this is normal django models behavior?).

Reply:
Yeah, Django will always convert `<FK_FIELD_NAME>` to `<FK_FIELD_NAME>_id` when making queries, and it can also read the `id` of a model instance that is assigned to `<FK_FIELD_NAME>_id`, which is probably why it works currently. I think I read somewhere that it's better to use the `id` than the full model instance, though, so that you don't have to keep the model instance in memory or something... I'm having trouble finding it in the Django docs, though.